tests/test_dereference.py

```diff
@@ -505,6 +523,21 @@ class Group(Document):
                 for m in group_obj.members:
                     self.assertIn('User', m.__class__.__name__)
 
+        # Delete one UserA instance so that there is
```

Comment:
same remark here, separate it into its own test method

Reply:
@bagerard Separated into its own test case in https://github.com/MongoEngine/mongoengine/pull/2007/commits/fdcaca42ae6145ac102ec3475bdc8c550a7a604a